docs/source/guides/dfs_with_dask_entitysets.rst

```diff
@@ -0,0 +1,102 @@
+DFS with Dask Entitysets
+========================
+Creating a feature matrix from a very large dataset can be problematic if the underlying pandas dataframes that make up the entities cannot easily fit in memory. To help get around this issue, Featuretools supports creating entities and entitysets from Dask dataframes. These entitysets can then be passed to ``ft.dfs`` to create a feature matrix, which will be returned as a Dask dataframe. In addition to working on larger than memory datasets, this approach also allows users to take advantage of the parallel processing capabilities offered by Dask.
```

Comment:
It looks like the rest of the documentation styles 'entityset' either as `EntitySet` or 'entity set' so we should probably mirror one of those here for consistency